[2025-04-05T15:55:33.722+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T15:55:33.769+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T15:55:32.315838+00:00 [queued]>
[2025-04-05T15:55:33.780+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T15:55:32.315838+00:00 [queued]>
[2025-04-05T15:55:33.781+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T15:55:33.798+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_coin_data> on 2025-04-05 15:55:32.315838+00:00
[2025-04-05T15:55:33.807+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'get_coin_data', 'manual__2025-04-05T15:55:32.315838+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpiye5rni7']
[2025-04-05T15:55:33.809+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask get_coin_data
[2025-04-05T15:55:33.810+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=484) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T15:55:33.812+0000] {standard_task_runner.py:63} INFO - Started process 509 to run task
[2025-04-05T15:55:33.832+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T15:55:33.886+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T15:55:32.315838+00:00 [running]> on host a4017603cc0b
[2025-04-05T15:55:34.025+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='get_coin_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T15:55:32.315838+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T15:55:32.315838+00:00'
[2025-04-05T15:55:34.026+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T15:55:34.220+0000] {logging_mixin.py:188} INFO - Error fetching data: 429 Client Error: Too Many Requests for url: https://api.coincap.io/v2/assets/bitcoin/history?interval=h1&start=1609459200000&end=1609545599999
[2025-04-05T15:55:34.222+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T15:55:34.224+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T15:55:34.240+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=get_coin_data, run_id=manual__2025-04-05T15:55:32.315838+00:00, execution_date=20250405T155532, start_date=20250405T155533, end_date=20250405T155534
[2025-04-05T15:55:34.271+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T15:55:34.309+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T15:55:34.311+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
